HowTo install Linux (Ubuntu) tools for C++ programming with Eclipse IDE/readme.md

```diff
@@ -45,10 +45,10 @@ tar -xzf eclipse-cpp-2019-09-R-linux-gtk-x86_64.tar.gz
 >
 > \> sudo mv eclipse /opt/
 
-Create a desktop launcher for Eclipse:
+Create a desktop launcher for Eclipse with the following content:
 
 ```sh
-sudo nano /usr/share/applications/eclipse.desktop
+sudo gedit ~Desktop/eclipse.desktop
```

Comment:
Why `~Desktop/eclipse.desktop` is better than `/usr/share/applications/eclipse.desktop`?